music21/abcFormat/testFiles.py

```diff
@@ -612,8 +676,116 @@ def testBasic(self):
             if title == 'D Fragment':
                 sharps = s.parts[0].keySignature.sharps
                 self.assertEqual(sharps, 7, 'C# key signature should be parsed as 7 sharps')
+                
+    def testAbc21(self):
+        from music21 import abcFormat, note
+        from music21.abcFormat import translate
+        from music21.musicxml import m21ToXml
 
+        af = abcFormat.ABCFile(abcVersion="2.1")
+
+        gex = m21ToXml.GeneralObjectExporter()
         
+        files = [carryThrough]
+        for i, tf in enumerate(files):
+            ah = af.readstr(tf)
+            title = ah.getTitle()
+            environLocal.printDebug([title])
+            s = translate.abcToStreamScore(ah)
+            try:
```

Comment:
are we worried about stream to musicxml conversion?  If not, cut this test which will slow down testing of what we actually want to test.

Reply:
Sorry. Just cutting, pasting, and not thinking. The gex stuff is gone now.